src/components/Filter/modules/ListFilter/ListFilter.tsx

```diff
@@ -47,16 +47,25 @@ const ListFilter = (
         }
     },
     getBrowserQueryUrlValue: value => value && Array.from(value),
-    getDefaultFilterValue: () => new Set(options.map(item => item.value)),
+    getDefaultFilterValue: () => listFilterOptions.initialValue ?
+        new Set([listFilterOptions.initialValue].flat()) :
+        new Set(options.map(item => item.value)),
+
     isDefaultFilterValue: value => {
-        if (value) {
```

Comment:
With the logic being modified here, there should be updated tests in the `listFilter.test.js` file to cover this.